librz/core/cmd/cmd_analysis.c

```diff
@@ -4832,24 +4832,32 @@ static void cmd_analysis_graph(RzCore *core, const char *input) {
 RZ_API int rz_core_analysis_refs(RzCore *core, const char *input) {
 	int cfg_debug = rz_config_get_b(core->config, "cfg.debug");
 	ut64 from, to;
-	int rad;
+	RzOutputMode out_mode;
```

Comment:
Could you extract this particular change, also maybe port `aar` command to the rzshell (https://github.com/rizinorg/rizin/issues/1442) in a separate PR?

Reply:
With extracting you mean as a separated commit?